ds_judgements_public_ui/sass/includes/_cards_homepage.scss

```diff
@@ -1,89 +1,208 @@
+    }
+
+    @media (max-width: $grid-breakpoint-extra-large) {
+      padding: $space-7;
     }
   }
 
   @media (min-width: $grid-breakpoint-small) {
     gap: 0;
   }
 
-  @include container;
+  @media (width <= 1199px) {
+    float: left;
+    flex-direction: column;
+    gap: 0 !important;
+  }
 }
 
 .card-homepage {
   display: flex;
-  gap: $space-4;
+  gap: $space-2;
+
   margin: $space-4 $space-8;
+  padding: 0;
+
+  text-decoration: underline;
+  text-decoration-color: colour-var("accent-background-light");
+
+  background-color: colour-var("accent-background-light");
+  border-top: 4px solid $color-yellow;
+
+  &:hover {
+    text-decoration: underline;
+    text-decoration-color: colour-var("accent-background");
+    background-color: colour-var("accent-background");
+  }
 
   h3 {
     margin-top: auto;
     margin-bottom: 0;
-    font-weight: $typography-bold-font-weight;
+
+    color: colour-var("link");
+    text-align: left;
+    text-decoration: underline !important;
+    text-decoration-color: colour-var("accent-background-light");
+    text-wrap: wrap;
 
     a {
       color: colour-var("link");
+      text-decoration: none;
     }
 
-    @media only screen and (max-width: $grid-breakpoint-medium) {
+    @media only screen and (width <= 1199px) {
       margin-top: $space-2;
 
-      font-size: $typography-md-text-size;
+      font-size: $typography-lg-text-size;
       line-height: $typography-sm-line-height;
       text-align: left;
+      text-decoration: underline;
+      text-decoration-color: colour-var("link");
       text-wrap: balance;
     }
   }
 
+  h3:visited {
+    color: colour-var("link");
+    text-decoration: underline;
+    text-decoration-color: colour-var("accent-background-light");
+  }
+
+  h3:hover {
+    color: colour-var("link");
+    text-decoration: none !important;
+  }
+
   &__legend-text {
     margin-top: $space-2;
     margin-bottom: $space-8;
-    text-wrap: balance;
+    color: colour-var("accent-font-base") !important;
+    text-align: left;
 
-    @media (max-width: $grid-breakpoint-medium) {
+    @media (width <= 1199px) {
       visibility: hidden;
     }
   }
 
   img {
     display: flex;
-    width: 100px;
-    height: 100px;
+    width: 122px;
+    height: 161px;
     object-fit: contain;
+
+    @media (width <= 1199px) {
+      display: none;
+    }
   }
 
   &__legend {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
+    margin-top: $space-3;
+    padding-right: $space-4;
+    padding-left: $space-4;
 
-    @media (max-width: $grid-breakpoint-medium) {
+    @media (width <= 1199px) {
       justify-content: flex-start;
 
       margin-top: $space-8;
       padding: 0;
 
       text-wrap: balance;
 
-      background-color: colour-var("accent-background-light");
+      background-color: transparent;
     }
   }
 
-  @media (max-width: $grid-breakpoint-medium) {
+  @media (width <= 1199px) {
     gap: $space-4;
+
+    min-width: 150%;
```

Comment:
This seems wrong - we shouldn't extend our widths to break out of the border box layout like this. Does this need styling to the parent change so this can flow properly?